docs/evm/cadence/batched-evm-transactions.md

```diff
@@ -0,0 +1,641 @@
+    }
+
+    pre {
+        self.coa.balance().inFLOW() >= self.mintCost:
+            "CadenceOwnedAccount holds insufficient FLOW balance to mint - "
+            .concat("Ensure COA has at least ".concat(self.mintCost.toString()).concat(" FLOW"))
+    }
+
+    execute {
+        /* Wrap FLOW in EVM as WFLOW */
+        //
+        // Encode calldata & set value
+        let depositCalldata = EVM.encodeABIWithSignature("deposit()", [])
+        let value = EVM.Balance(attoflow: 0)
+        value.setFLOW(flow: self.mintCost)
+        // Call the WFLOW contract, wrapping the sent FLOW
+        let wrapResult = self.coa.call(
+            to: self.wflowAddress,
+            data: depositCalldata,
+            gasLimit: 15_000_000,
+            value: value
+        )
+        assert(
+            wrapResult.status == EVM.Status.successful,
+            message: "Wrapping FLOW as WFLOW failed: ".concat(wrapResult.errorMessage)
+        )
+
+        /* Approve the ERC721 address for the mint amount */
+        //
+        // Encode calldata approve(address,uint) calldata, providing the ERC721 address & mint amount
+        let approveCalldata = EVM.encodeABIWithSignature(
+                "approve(address,uint256)",
+                [self.erc721Address, UInt256(1_000_000_000_000_000_000)]
+            )
+        // Call the WFLOW contract, approving the ERC721 address to move the mint amount
+        let approveResult = self.coa.call(
+            to: self.wflowAddress,
+            data: approveCalldata,
+            gasLimit: 15_000_000,
+            value: EVM.Balance(attoflow: 0)
+        )
+        assert(
+            approveResult.status == EVM.Status.successful,
+            message: "Approving ERC721 address on WFLOW contract failed: ".concat(approveResult.errorMessage)
+        )
+
+        /* Attempt to mint ERC721 */
+        //
+        // Encode the mint() calldata
+        let mintCalldata = EVM.encodeABIWithSignature("mint()", [])
+        // Call the ERC721 contract, attempting to mint
+        let mintResult = self.coa.call(
+            to: self.erc721Address,
+            data: mintCalldata,
+            gasLimit: 15_000_000,
+            value: EVM.Balance(attoflow: 0)
+        )
+        // If mint fails, all other actions in this transaction are reverted
+        assert(
+            mintResult.status == EVM.Status.successful,
+            message: "Minting ERC721 token failed: ".concat(mintResult.errorMessage)
+        )
+    }
+}
+
+```
+</details>
+
+You can run the transaction at the following link using the community-developed Flow Runner tool:
+[`wrap_and_mint.cdc`](https://run.dnz.dev/snippet/4fe0d43fdaf968b9).
+
+This transaction takes two arguments:
+- WFLOW contract address: `0xd3bF53DAC106A0290B0483EcBC89d40FcC961f3e`
+- MaybeMintERC721 contract address: `0x2E2Ed0Cfd3AD2f1d34481277b3204d807Ca2F8c2`
+
+Before running, ensure that the network section - bottom left corner - displays Testnet. If not, click and select
```

Comment:
Bottom right?

Reply:
Good catch, got my directions swapped

Reply:
Fixed in [c19893b](https://github.com/onflow/docs/pull/983/commits/c19893b7ade1cd95602c6ab00f9c81bd77b149b4)